src/i18n/es.json

```diff
@@ -30,5 +30,8 @@
     "message": "Mensaje",
     "send": "Enviar",
     "cancel": "Cancelar"
+  },
+  "validation_messages": {
+    "error_required_field": "¡Ouch! El campo está vacío."
```

Comment:
What do you think about these names? thoughts?

Reply:
yes, this it's generic